agile-development/team-agreements/working-agreements/readme.md

```diff
@@ -14,7 +14,7 @@ their own, and adjust times, communication channels, branch naming policies etc.
 - We work as one team towards a common goal and clear scope
 - We make sure everyone's voice is heard, listened to
 - We show all team members equal respect
-- We pair-program as often as possible
+- We work as a team to have common expectations for technical delivery that are documented in a [Team Manifesto](../team-manifesto/readme.md).
```

Comment:
From within the Team manifesto I suggest linking to the [collaboration/pair programming page](https://github.com/microsoft/code-with-engineering-playbook/tree/master/agile-development/collaboration)